tests/test_runner/tests.py

```diff
@@ -203,8 +203,6 @@ def test_ticket_17477(self):
 
 class Sqlite3InMemoryTestDbs(TestCase):
 
-    available_apps = []
```

Comment:
I see the exception if it's removed:
```
======================================================================
ERROR: test_transaction_support (test_runner.tests.Sqlite3InMemoryTestDbs)
Ticket #16329: sqlite3 in-memory test databases
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/tim/code/django/django/apps/registry.py", line 148, in get_app_config
    return self.app_configs[app_label]
KeyError: 'auth'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/tim/code/django/tests/test_runner/tests.py", line 227, in test_transaction_support
    DiscoverRunner(verbosity=0).setup_databases()
  File "/home/tim/code/django/django/test/runner.py", line 537, in setup_databases
    self.parallel, **kwargs
  File "/home/tim/code/django/django/test/utils.py", line 205, in setup_databases
    call_command('check')
  File "/home/tim/code/django/django/core/management/__init__.py", line 130, in call_command
    return command.execute(*args, **defaults)
  File "/home/tim/code/django/django/core/management/base.py", line 345, in execute
    output = self.handle(*args, **options)
  File "/home/tim/code/django/django/core/management/commands/check.py", line 68, in handle
    fail_level=getattr(checks, options['fail_level']),
  File "/home/tim/code/django/django/core/management/base.py", line 374, in check
    include_deployment_checks=include_deployment_checks,
  File "/home/tim/code/django/django/core/management/base.py", line 361, in _run_checks
    return checks.run_checks(**kwargs)
  File "/home/tim/code/django/django/core/checks/registry.py", line 81, in run_checks
    new_errors = check(app_configs=app_configs)
  File "/home/tim/code/django/django/contrib/auth/checks.py", line 16, in check_user_model
    cls = apps.get_model(settings.AUTH_USER_MODEL)
  File "/home/tim/code/django/django/apps/registry.py", line 195, in get_model
    return self.get_app_config(app_label).get_model(model_name.lower())
  File "/home/tim/code/django/django/apps/registry.py", line 155, in get_app_config
    raise LookupError(message)
LookupError: No installed app with label 'auth'.
```
We should check if this is the proper fix or if this reveals some other problem, I'm not sure offhand.